appserver/tests/payara-samples/samples/clustered-singleton/clustered-singleton-test/src/main/java/com/flowlogix/clust/singleton/ClusteredSingletonEjbXml.java

```diff
@@ -37,24 +37,25 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-package fish.payara.nucleus.hazelcast;
+package com.flowlogix.clust.singleton;
```

Comment:
Same recommendations as the other PR. Package needs changing, and to Lombok or not to Lombok

Reply:
This will be fixed with the other PR, as you said :)